packages/ui/src/components/Alert/alert.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Alert as DefaultAlert, AlertProps as DefaultAlertProps } from 'antd'
+
+type CondoAlertProps = {
+    type: 'success' | 'info' | 'warning' | 'error';
```

Comment:
Why do you need to override type with same values?)

Reply:
because in DefaultAlertProps the type property is optional, and it seemed to me that it would be better if in the storybook this property was specified as mandatory:
<img width="978" alt="Screenshot 2022-12-06 at 13 37 29" src="https://user-images.githubusercontent.com/93817911/205861563-ebcf5541-8c55-4107-8f9c-589e56dea8f5.png">

or can it be done differently without override type with same values?)

Reply:
Oh....I see 👍 

Then your solution is ok...
Also solution with `type: DefaultAlertProps['type']` can be used to override field obligation